src/main/java/com/epam/ta/reportportal/core/dashboard/impl/UpdateDashboardHandlerImpl.java

```diff
@@ -126,10 +126,10 @@ public OperationCompletionRS updateDashboard(ReportPortalUser.ProjectDetails pro
 	public OperationCompletionRS addWidget(Long dashboardId, ReportPortalUser.ProjectDetails projectDetails, AddWidgetRq rq,
 			ReportPortalUser user) {
 		Dashboard dashboard = getShareableDashboardHandler.getAdministrated(dashboardId, projectDetails);
-		BusinessRule.expect(dashboard.getDashboardWidgets().size() > MAX_WIDGET_ON_DASHBOARD, Predicates.equalTo(true)).verify(
+		BusinessRule.expect(dashboard.getDashboardWidgets().size() >= MAX_WIDGET_ON_DASHBOARD, Predicates.equalTo(false)).verify(
 				ErrorType.DASHBOARD_UPDATE_ERROR,
 				Suppliers.formattedSupplier(
-						"The limit of 300 dashboards has been reached. To create a new one you need to delete at least one created previously")
+						"The limit of 300 dashboards has been reached. To create a new one you need to delete at least one created previously.")
```

Comment:
Please, change "300" in error message to placeholder for amount, and fill it with MAX_WIDGET_ON_DASHBOARD.

Reply:
done